### PR TITLE
Document how to extract inline SVGs for strict CSP

### PR DESCRIPTION
**Pull request by charlesroelli**
Cf. https://github.com/twbs/bootstrap/issues/25394, https://github.com/twbs/bootstrap/issues/33134

### CODE REVIEW

File: site/content/docs/5.1/customize/overview.md [hunk 1]
```diff
@@ -49,3 +49,24 @@ Several Bootstrap components include embedded SVGs in our CSS to style component
...
+        generator: {
+          filename: 'icons/[hash].svg'
+        },
+      },
+      ...
+    ],
+  },
+```
+
+This rule will place inline SVGs into files under the `icons/` directory of the build directory, with the filename based on the hash of the file contents.  This should allow you to use a strict CSP that does not allow inline `data:` URLs in stylesheets.
```
Comment (ffoodd) [inline site/content/docs/5.1/customize/overview.md:72]:
```suggestion
This rule will place inline SVGs into files under the `icons/` directory of the build directory, with the filename based on the hash of the file contents. This should allow you to use a strict CSP that does not allow inline `data:` URLs in stylesheets.
```

File: site/content/docs/5.1/customize/overview.md [hunk 2]
```diff
@@ -49,3 +49,24 @@ Several Bootstrap components include embedded SVGs in our CSS to style component
 - [Navbar toggle buttons]({{< docsref "/components/navbar#responsive-behaviors" >}})
 
 Based on [community conversation](https://github.com/twbs/bootstrap/issues/25394), some options for addressing this in your own codebase include replacing the URLs with locally hosted assets, removing the images and using inline images (not possible in all components), and modifying your CSP. Our recommendation is to carefully review your own security policies and decide on the best path forward, if necessary.
+
```
Comment (ffoodd) [inline site/content/docs/5.1/customize/overview.md:52]: A heading is needed here to introduce this new section.

Something like:

`### Workaround using Webpack`

**Review by ffoodd — CHANGES_REQUESTED** [review]
This is great, thanks a lot. I'm still wondering if it shouldn't be moved to the Webpack page, and simply linked to from the CSP section?

Any thought?

**Comment by scallywag** [comment]
Hi, thanks for the review! Please see the updated changes.

**Comment by mdo** [comment]
This will need to be updated given the new Webpack guide we shipped in v5.2.0. Any chance you want to take a crack at that?

**Comment by charlesroelli** [comment]
Sure, I will have a go at it.

**Pull request closed without merging**